packages/calcite-components/src/custom-theme/button.ts

```diff
@@ -0,0 +1,17 @@
+import { html } from "../../support/formatting";
+
+const buttonProps: [args: string, content: string] = ["", ""];
+
+export const button = (props: { kind?: string; appearance?: string }): string => {
+  const [buttonArgs, content] = Object.entries(props)
+    .filter((a) => a && a[0] && a[1] && a[1] !== "")
```

Comment:
Looking better! 😎

One note, since `a` (`[key, value]`) is guaranteed, I think this can be simplified to `.filter(([key, value]) => key && value && value !== "")`.

Thanks for making these tweaks!

Reply:
this is what I get for quick copy-paste. 🤦🏻 
thanks for the catch!